src/app/helpers/controller/cms-mixin.js

```diff
@@ -98,31 +98,22 @@ export async function fetchPageData({slug, preserveWrapping, setsPageTitleAndDes
 
 export function usePageData(fpdParams) {
     const [pageData, setPageData] = useState();
-    const [pageDataError, setPageDataError] = useState();
     let statusPage = null;
 
     useEffect(() => {
-        async function fetchData() {
-            try {
-                setPageData(await fetchPageData(fpdParams));
-            } catch (err) {
-                setPageDataError(err);
-            }
-        }
-        fetchData();
+        fetchPageData(fpdParams).then(setPageData);
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [fpdParams.slug]);
```

Comment:
just in case you didn't know about it, there's also `eslint-disable-line`. I like using that for lines that are short because then it'll be deleted along with the line if it's removed:
`}, [fpdParams.slug]); // eslint-disable-line react-hooks/exhaustive-deps`